pubspec.yaml

```diff
@@ -1,13 +1,11 @@
 name: ffi
-version: 0.3.0-nullsafety.2
+version: 1.0.0-nullsafety.0
```

Comment:
We'll need an actual 1.0.0 stable release soon also. Is this 1.0.0 pre-release to ensure things are working first?

Reply:
Done